internal/report/report.go

```diff
@@ -1165,7 +1165,7 @@ func ProfileLabels(rpt *Report) []string {
 
 // reportLabels returns printable labels for a report. Includes
 // profileLabels.
-func reportLabels(rpt *Report, g *graph.Graph, origCount, droppedNodes, droppedEdges int, fullHeaders bool) []string {
+func reportLabels(rpt *Report, g *graph.Graph, origCount, droppedNodes, droppedEdges int, fullHeaders bool, helperURL bool) []string {
```

Comment:
It looks like fullHeaders and helperURL always have the same value. And it makes sense to me that the helper URL might be required only when full headers are required. 

Perhaps no argument needs to be added to this function, and fullHeaders can be used in place of helperURL?

Reply:
Done.